examples/darwin-framework-tool/BUILD.gn

```diff
@@ -74,9 +75,12 @@ executable("darwin-framework-tool") {
     # framework" library, because otherwise the weak symbols from
     # RequiredPrivilege.cpp mean we never actually pull in the function from
     # privilege-storage.cpp.
```

Comment:
Do we still actually need privilege-storage here, if we're no longer linking to libCHIP directly?  I don't think we do.

Reply:
No, we do not. I will remove it. Thanks

Reply:
Removed.